docs/actions.md

```diff
@@ -76,13 +78,14 @@ Action | Description
 <a href="/actions/xcode_install">xcode_install</a> | Make sure a certain version of Xcode is installed
 <a href="/actions/verify_pod_keys">verify_pod_keys</a> | Verifies all keys referenced from the Podfile are non-empty
 <a href="/actions/xctest">xctest</a> | Runs tests on the given simulator
-<a href="/actions/build_app">build_app</a> | Alias for the `build_ios_app` action
```

Comment:
What's up with the changes in action order here on each PR by the way?

Reply:
I actually don't know. We used to sort them by popularity, but we can't any more due to changed metrics 😢